src/bin/gifski.rs

```diff
@@ -32,9 +32,9 @@ use std::thread;
 use std::time::Duration;
 
 #[cfg(feature = "video")]
-const VIDEO_FRAMES_ARG_HELP: &'static str = "one MP4/WebM video, or multiple PNG animation frames";
+const VIDEO_FRAMES_ARG_HELP: &'static str = "one video file supported by FFmpeg, or multiple PNG image files";
```

Comment:
Notably, it is now possible to convert from GIF to GIF by adjusting its speed and size (though there could be quality losses).